manifest.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: matrixdeployment
+  name: matrixdeployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: matrixdeploymentpod
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: matrixdeploymentpod
+    spec:
+      containers:
+      - name: matrixcontainer
+        image: sbacr.azurecr.io/sbimage:latest
+        resources: {}
+        ports:
+          - containerPort: 8080
+status: {}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  name: matrixservice
+spec:
+  type: ClusterIP
+  ports:
+  - port: 8080
+    targetPort: 8080
+  selector:
+    app: matrixdeploymentpod
+status:
+  loadBalancer: {}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingressservice
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /
+spec:
+  rules:
+    - http:
+        paths:
+        - pathType: Prefix
+          path: "/"
+          backend:
+            service:
+              name: matrixservice
+              port: 
+                number: 8080
+
+# To run:
+# `kubectl apply -f manifest.yaml`
+# http://20.72.217.18/rows/3/columns/4
+
+# Troubleshoot/Confirm everything is running with:
+# `kubectl get pods`
+# `kubectl describe pod`
+# `kubectl logs <podname>`
+# `kubectl get deployment`
+# `kubectl get replicaset`
+# `kubectl get service`
+# `kubectl describe service`
+# `kubectl get ingress` - gives you IP address
+# `kubectl describe ingress`
```

Comment:
The README is a good place to put anything related to running the service, troubleshooting and testing, you should probably move this there :)

Reply:
Makes sense. Removed from here for now while I sort out my README, but saved in my notes so that I can add reference later when putting together that doc